src/page-modules/contact/ticket-control/forms/feeComplaintForm.tsx

```diff
@@ -16,423 +16,425 @@ type FeeComplaintFormProps = {
   send: (event: typeof ticketControlFormEvents) => void;
 };
 
-export const FeeComplaintForm = ({ state, send }: FeeComplaintFormProps) => {
-  const { t } = useTranslation();
+type FirstAgreementProps = FeeComplaintFormProps;
```

Comment:
I think you should use TypeScript Pick here so that it's possible to add properties to FeeComplaintForm and not FirstAgreementProps